<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserLearns</class>
 <widget class="QDialog" name="UserLearns">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>888</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Learning session</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QTextEdit" name="taskTextEdit">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="userTextEdit">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="statsButton">
         <property name="text">
          <string/>
         </property>
         <property name="shortcut">
          <string>Ctrl+S</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="restartButton">
         <property name="text">
          <string/>
         </property>
         <property name="shortcut">
          <string>Ctrl+R</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="finishButton">
         <property name="text">
          <string/>
         </property>
         <property name="shortcut">
          <string>Ctrl+F</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="showtasksButton">
         <property name="text">
          <string/>
         </property>
         <property name="shortcut">
          <string>Ctrl+T</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="nextButton">
         <property name="text">
          <string/>
         </property>
         <property name="shortcut">
          <string>Ctrl+Return</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="reloadDataButton">
         <property name="text">
          <string/>
         </property>
         <property name="shortcut">
          <string>Ctrl+Shift+R</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="helpButton">
         <property name="text">
          <string/>
         </property>
         <property name="shortcut">
          <string>Ctrl+H</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="clearButton">
         <property name="text">
          <string/>
         </property>
         <property name="shortcut">
          <string>Ctrl+Shift+D</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="fontUpButton">
         <property name="text">
          <string/>
         </property>
         <property name="shortcut">
          <string>Ctrl+U</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="fontDownButton">
         <property name="text">
          <string/>
         </property>
         <property name="shortcut">
          <string>Ctrl+D</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="wordsNumberComboBox"/>
       </item>
       <item>
        <spacer name="modeSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>13</width>
           <height>17</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="current_languageLabel">
         <property name="text">
          <string>lang</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="engModeRadioButton">
         <property name="text">
          <string>eng</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="sweModeRadioButton">
         <property name="text">
          <string>swe</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QProgressBar" name="progressBar">
         <property name="value">
          <number>24</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>18</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="resultLabel">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
